generated/nifake/nifake.proto

```diff
@@ -94,17 +94,17 @@ enum Color {
   COLOR_BLACK = 42;
 }
 
-enum FloatEnum {
-  FLOAT_ENUM_UNSPECIFIED = 0;
+enum FloatEnumMapped {
+  FLOAT_ENUM_MAPPED_UNSPECIFIED = 0;
```

Comment:
Yes, MAPPED substring in UNSPECIFIED constant is intentional because of special case of `*AttributeValues` enums. I have replied to your comment in `proto_helpers.mako` with more details.